gfx/webrender/webrender.h

```diff
@@ -117,5 +117,10 @@ WR_FUNC;
 
 #undef WR_FUNC
 
+// The pointer returned by wr_readback_buffer must be freed by rust, not C
+// After using the data, it is the responsibility of the caller to free the memory
+// by giving the pointer and out_length to wr_free_buffer.
+const uint8_t* wr_readback_buffer(uint32_t width, uint32_t height, uint32_t* out_length, uint32_t* out_capacity);
+void wr_free_buffer(const uint8_t* pointer, uint32_t length, uint32_t capacity);
```

Comment:
You should still swaddle these signatures in WR_INLINE/WR_FUNC unless you're going to guard all the callsites with MOZ_ENABLE_WEBRENDER.